spec/models/miq_report_result_spec.rb

```diff
@@ -11,12 +11,10 @@
     @group = FactoryGirl.create(:miq_group)
     @user  = FactoryGirl.create(:user, :miq_groups => [@group])
 
-    5.times do |i|
-      vm = FactoryGirl.build(:vm_vmware)
-      vm.evm_owner_id = @user.id           if i > 2
-      vm.miq_group_id = @user.current_group.id if vm.evm_owner_id || (i > 1)
-      vm.save
-    end
+    @vm = FactoryGirl.build(:vm_vmware)
+    @vm.evm_owner_id = @user.id
+    @vm.miq_group_id = @user.current_group.id if @vm.evm_owner_id
+    @vm.save
 
```

Comment:
this refactoring probably doesn't belong in this PR anymore. @gtanzillo do you feel strongly either way?